test/h2_worker_test.exs

```diff
@@ -439,6 +439,44 @@ defmodule Sparrow.H2WorkerTest do
     end
   end
 
+  test "server correctly starting with succesfull connection and does not scheduale or runs pinging",
+       context do
+    ptest [
+            domain: string(min: 3, max: 10, chars: ?a..?z),
+            port: int(min: 0, max: 65_535),
+            tls_options: list(of: atom(), min: 0, max: 3)
+          ],
+          repeat_for: @repeats do
+      ponger = pid()
+      ping_interval = nil
+
+      with_mock H2Adapter,
+        open: fn _, _, _ -> {:ok, context[:connection_ref]} end,
+        ping: fn _ ->
+          send(self(), {:PONG, ponger})
```

Comment:
Where do we wait for this message to actually come? I can see we check if function was called

Reply:
It is not needed. I'll remove it.